global/meta/models.yml

```diff
@@ -471,6 +447,19 @@ meeting_user:
   comment:
     type: HTMLStrict
     restriction_mode: A
+  number:
+    type: string
+    restriction_mode: A
+  structure_level:
+    type: string
+    restriction_mode: A
+  about_me:
+    type: HTMLStrict
+    restriction_mode: B
```

Comment:
Choose a good place to document B for Oskar or describe it here or invent a new tag in models.yml (may be the best option. if description is used anywhere)

Reply:
It is documented in the Action wiki, that is a good place to look at.

Reply:
The action wiki is not a good place for the description. I talked to @rrenkert and @r-peschke  about it and added the description for now to the model.yml.

Reply:
The action wiki is not a good place, because
- the action wiki permissions are write-permissions, the restriction modes are read-permissions
- there is no relation between the used abbreviations/group letters between backend actions and autoupdate restriction modes
- one field may appear in various actions. Would be difficult to find them all to have an idea, what a specific restriction mode is for.

Reply:
The place would be the documentation of all other restriction modes, the go files in the AU repo. To get the information to @ostcar and through that in the file, the usual way to go would be an issue in the according repo where the needed information is documented to be transferred into the file. We should remove this information from the models.yml again.

Reply:
Since I currently don't have a branch for this change, a Issue would be the best place.